karaoke.py

```diff
@@ -198,16 +200,16 @@ def render_splash_screen(self):
             if (not self.hide_ip):
                 p_image = pygame.image.load(self.generate_qr_code())
                 p_image = pygame.transform.scale(p_image, (150, 150))
-                self.screen.blit(p_image, (0,0))
```

Comment:
are you finding that there are screen cropping issues? Is it possible you have a miscalibrated screen?

Reply:
Maybe it was just my screen.